include/rmlab/tool/dump.cpp

```diff
@@ -0,0 +1,96 @@
+/* Copyright 2017-2018 Axel Huebl
```

Comment:
Please feel free to add your name to this line :)
(Is "pure ASCII" ok for you? Had bad experiences with some compilers with non-ascii comments in source files.)